honeybee_schema/energy/hvac/allair.py

```diff
@@ -0,0 +1,180 @@
+    vav_dcw_base = 'VAV district chilled water with no reheat with baseboard electric'
+    vav_dcw_guh = 'VAV district chilled water with no reheat with gas unit heaters'
+    vav_dcw_zhp = 'VAV district chilled water with no reheat with zone heat pump'
+
+
+class PVAVEquipmentType(str, Enum):
+    pvav_gbr = 'PVAV with gas boiler reheat'
+    pvav_ashp = 'PVAV with central air source heat pump reheat'
+    pvav_dhw = 'PVAV with district hot water reheat'
+    pvav_pfp = 'PVAV with PFP boxes'
+    pvav_ger = 'PVAV with gas heat with electric reheat'
+
+
+class PSZEquipmentType(str, Enum):
+    psz_e_base = 'PSZ-AC with baseboard electric'
+    psz_gb_base = 'PSZ-AC with baseboard gas boiler'
+    psz_dhw_base = 'PSZ-AC with baseboard district hot water'
+    psz_guh = 'PSZ-AC with gas unit heaters'
+    psz_ec = 'PSZ-AC with electric coil'
+    psz_gc = 'PSZ-AC with gas coil'
+    psz_gb = 'PSZ-AC with gas boiler'
+    psz_ashp = 'PSZ-AC with central air source heat pump'
+    psz_dhw = 'PSZ-AC with district hot water'
+    psz_ac = 'PSZ-AC with no heat'
+    psz_dcw_e_base = 'PSZ-AC district chilled water with baseboard electric'
+    psz_dcw_gb_base = 'PSZ-AC district chilled water with baseboard gas boiler'
+    psz_dcw_dhw_base = 'PSZ-AC district chilled water with baseboard district hot water'
+    psz_dcw_guh = 'PSZ-AC district chilled water with gas unit heaters'
+    psz_dcw_ec = 'PSZ-AC district chilled water with electric coil'
+    psz_dcw_gc = 'PSZ-AC district chilled water with gas coil'
+    psz_dcw_gb = 'PSZ-AC district chilled water with gas boiler'
+    psz_dcw_ashp = 'PSZ-AC district chilled water with central air source heat pump'
+    psz_dcw_dhw = 'PSZ-AC district chilled water with district hot water'
+    psz_dcw_ac = 'PSZ-AC district chilled water with no heat'
+    psz_hp = 'PSZ-HP'
+
+
+class PTACEquipmentType(str, Enum):
+    ptac_e_base = 'PTAC with baseboard electric'
+    ptac_gb_base = 'PTAC with baseboard gas boiler'
+    ptac_dhw_base = 'PTAC with baseboard district hot water'
+    ptac_guh = 'PTAC with gas unit heaters'
+    ptac_ec = 'PTAC with electric coil'
+    ptac_gc = 'PTAC with gas coil'
+    ptac_gb = 'PTAC with gas boiler'
+    ptac_ashp = 'PTAC with central air source heat pump'
+    ptac_dhw = 'PTAC with district hot water'
+    ptac = 'PTAC with no heat'
+    pthp = 'PTHP'
+
+
+class FurnaceEquipmentType(str, Enum):
+    furnace = 'Forced air furnace'
+
+
+class VAV(_AllAirBase):
+    """Variable Air Volume (VAV) HVAC system."""
+
+    type: constr(regex='^VAV$') = 'VAV'
+
+    equipment_type: VAVEquipmentType = Field(
+        VAVEquipmentType.vav_chill_gbr,
+        description='Text for the specific type of the system and equipment '
+        'as it appears in the OpenStudio standards gem.'
+    )
+
+
+class PVAV(_AllAirBase):
+    """Packaged Variable Air Volume (PVAV) HVAC system."""
+
+    type: constr(regex='^PVAV$') = 'PVAV'
+
+    equipment_type: PVAVEquipmentType = Field(
```

Comment:
Is there some sort of documentation for these types? For beginner users it will be helpful to add more info about what it VAV versus PVAV and for advanced users it will be helpful to have an idea about the default values.

Also are we going to expose any of  the default values from the schema or we consider the template as read-only.

Reply:
There's ASHRAE 90.1.  I don't think it really helps anyone to paste whole sections of the standard here or list the hundreds of default values that are being set by it but I can just add "more info can be found in ASHRAE 90.1."

Is there anything else you are thinking of beyond this?

Reply:
I think it will be helpful if we can provide a link to the document. That doesn't mean that we have to "list the hundreds of default values that are being set by it" - What about adding this link? (https://ashrae.iwrapper.com/ViewOnline/Standard_90.1-2019). If we can note the exact section/chapter in the document that will be even better.

Reply:
That link does not work:
![image](https://user-images.githubusercontent.com/5567574/89468430-6f67a480-d745-11ea-8658-3615f07cbc13.png)

... and this kinda illustrates the problem with links. They may not always work or they may change. What if we just reference the standard itself in a `Note:` section?

Like what we do with the references in ladybug-comfort:
https://www.ladybug.tools/ladybug-comfort/docs/ladybug_comfort.pmv.html

Reply:
To clarify further, my proposal is to keep using honeybee-energy docstrings as the place to put all of the detailed references and let the schema docs just be schema docs.

Reply:
A Note section would be good. This link should always work: (https://www.ashrae.org/technical-resources/standards-and-guidelines/read-only-versions-of-ashrae-standards). Apparently, the other link only works if clicked from this page. I don't think that because a link might be broken in the future we should not include links! We can update the link when it is broken.

>  my proposal is to keep using honeybee-energy docstrings as the place to put all of the detailed references and let the schema docs just be schema docs.

I think the schema should work as a self-contained entity. Adding documentation to honeybee-energy is good but doesn't make up for the schema.